src/main.rs

```diff
@@ -25,77 +19,9 @@ struct Opts {
 fn main() -> Result<()> {
     let opts: Opts = Opts::parse();
 
-    let engine = Engine::default();
-    let module = Module::from_file(&engine, &opts.script)
-        .map_err(|e| anyhow!("Couldn't load script {:?}: {}", &opts.script, e))?;
+    let statistics = run(opts.script, opts.input)?;
```

Comment:
Nit: `run` returns `Result<ExecutionResult>` so `execution_result` might be a better variable name here.

Reply:
But based on [my comment](https://github.com/Shopify/script-runner/pull/15#discussion_r882823526), we should call it `run_result` or `function_run_result` instead.